src/domain/phone-provider/index.types.d.ts

```diff
@@ -0,0 +1,31 @@
+import { UnknownPhoneProviderServiceError } from "@domain/errors"
+import { PhoneNumberInstance } from "twilio/lib/rest/lookups/v1/phoneNumber"
```

Comment:
We shouldn't have imports like this in declaration files (see here)
```suggestion
type PhoneProviderServiceError = import("./errors").PhoneProviderServiceError
type UnknownPhoneProviderServiceError = import("@domain/errors").UnknownPhoneProviderServiceError
```